gpio-falcon: make sure the default irq trigger mode is set to IRQ_TYPE_LEVEL_LOW

--- a/drivers/gpio/gpio-falcon.c
+++ b/drivers/gpio/gpio-falcon.c
@@ -218,8 +218,7 @@ static int falcon_gpio_irq_map(struct ir
 				      handle_simple_irq, port->name);
 	irq_set_chip_data(irq, port);
 
-	/* set to negative logic (falling edge, low level) */
-	port_w32_mask(port, 0, 1 << hw, GPIO_EXINTCR0);
+	irq_set_irq_type(irq, IRQ_TYPE_LEVEL_LOW);
 	return 0;
 }
 
